tutor/src/screens/my-courses/dashboard/offering-block.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useCallback, useEffect, ReactElement, Dispatch, SetStateAction } from 'react'
+import { map, filter } from 'lodash'
+import moment from 'moment'
+import { Icon } from 'shared'
+import UiSettings from 'shared/model/ui-settings'
+import Tabs from '../../../components/tabs'
+import CourseInformation from '../../../models/course/information'
+import { useNumberOfStudents } from '../../../store/courses'
+import CreateACourse from './create-course'
+import CoursePreview from './preview-course'
+import ViewCourse from './view-course'
+import Resource from './resource'
+
+import { Offering, Course, ID } from '../../../store/types'
+
+const isCourseCurrent = (course: Course) => moment().isBefore(course.ends_at)
+const isCoursePast = (course: Course) => moment().isAfter(course.ends_at)
+
+const sortByCourseEndsAt = (courseA: Course, courseB: Course) => {
+    if(moment(courseA.ends_at).isAfter(courseB.ends_at)) { return 1 }
+    if(moment(courseA.ends_at).isBefore(courseB.ends_at)) { return -1 }
+     return 0
+}
+const sortCurrentCourses = (courses: Course[]) => courses.sort((a, b) => {
+    // no students courses put them at the end of the list
+    if (useNumberOfStudents(a.id) === 0) { return -1 }
+    return sortByCourseEndsAt(a, b);
+})
+const sortPastCourses = (courses: Course[]) => courses.sort((a, b) => {
+    return sortByCourseEndsAt(a, b);
+})
+
+interface ResourcesInfoProps {
+    offering: Offering
+    os_book_id: string
+    isFirstBlock: boolean
+    renderCoursePreview: () => ReactElement | null
+}
+
+/**
+ * Component that displays the resources
+ */
+const ResourcesInfo: React.FC<ResourcesInfoProps> = ({ offering, os_book_id, isFirstBlock, renderCoursePreview }) => {
+    const generalResources = 
+    <>
+        <Resource
+          title="Instructor Getting Started Guide"
+          info="Find information on OpenStax Tutor features and answers to common questions"
+          link={CourseInformation.gettingStartedGuide.teacher} />
+        <Resource
+          title={<span><Icon type="play-circle"/> Video Tutorials </span>}
+          info="Step by step instructions on some of the most important tasks in OpenStax Tutor"
+          link={CourseInformation.videoTutorials} />
+    </>
+    return (
+    <>
+        {renderCoursePreview()}
+        {isFirstBlock && generalResources}
```

Comment:
instead of always building `generalResources` then conditionally rendering it, you could do:
```ts
const generalResources = isFirstBlock && (
   <>
      <Resource .../>
      <Resource .../>
  </>
)
```

and then include whatever `generalResources` got set to here

Reply:
👍